interface/backend/api/views.py

```diff
@@ -37,16 +41,34 @@ class ImageAvailableApiView(APIView):
 
     def get(self, request):
         """
-        Return a list of files and folders in dataset in the form
+        Return a sorted(by name) list of files and folders 
+        in dataset in the form
         {'directories': [
             {
                 'name': directory_name1,
                 'children': [ file_name1, file_name2, ... ]
             }, ... ]
         }
-
+        
+        TODO Dynamically fetch deeper directories
         """
-        return JsonResponse({'directories': []})
+        # example source:
+        # /images/LIDC-IDRI-0001/1.3.6.1.4.1.14519.5.2.1.6279.6001.298806137288633453246975630178
+        src_dir = os.environ.get('IMAGE_SOURCE', '/images')
+        fss = FileSystemStorage(src_dir)
+        list_dirs = fss.listdir(src_dir)
+        dirs = sorted(list_dirs[0])
+        tree = []
+        if dirs:
```

Comment:
Unnecessary? I mean, if we loop over an empty ``dirs`` the loop will just not run..